docs/Readme.md

```diff
@@ -1,56 +1,28 @@
 # Knora Documentation
 
-## 1 Paradox Documentation
+## 1 MkDocs Documentation
```

Comment:
This file still talks about Paradox and Jekyll, shouldn't that part be deleted? Also, the sections are numbered 1, 4, 5, 6.

Reply:
Thanks. Now there should be no more mention of paradox including any links.